source/routing/raptor.h

```diff
@@ -201,7 +201,9 @@ struct RAPTOR {
                             const nt::RTLevel rt_level,
                             const type::VehicleJourney* vj,
                             const uint16_t l_zone,
-                            DateTime workingDate);
+                            DateTime workingDate,
+                            DateTime working_walking_duration,
```

Comment:
```suggestion
                            DateTime working_fallback_duration,
```

Reply:
Done in 3789955136b424900bec1d4cea2b932e3d6bb971